cdm/core/src/main/java/ucar/nc2/dataset/VariableDS.java

```diff
@@ -860,6 +865,10 @@ private void createEnhancements() {
     if (standardizerAtt != null && this.enhanceMode.contains(Enhance.ApplyStandardizer) && dataType.isFloatingPoint()) {
       this.standardizer = Standardizer.createFromVariable(this);
     }
+    Attribute normalizerAtt = findAttribute(CDM.NORMALIZE);
+    if (normalizerAtt != null && this.enhanceMode.contains(Enhance.ApplyStandardizer) && dataType.isFloatingPoint()) {
```

Comment:
Think this should be
```suggestion
    if (normalizerAtt != null && this.enhanceMode.contains(Enhance.ApplyNormalizer) && dataType.isFloatingPoint()) {
```

Reply:
Absolutely!